app/components/TracksCard/index.js

```diff
@@ -4,37 +4,209 @@
  *
  */
 
-import React from 'react';
+import React, { useRef, useState } from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
 import { Card, Image, Button } from 'antd';
 import { isEmpty } from 'lodash';
 import If from '@components/If';
+import { colors } from '@themes';
+// import { useHistory } from 'react-router-dom';
 
 const CustomCard = styled(Card)`
   && {
-    margin: 20px 0;
+    border-radius: 0.5rem;
+    padding: 0.5rem;
     width: 15rem;
-    height: 37rem;
+    height: 22rem;
+    border: none;
+    background-color: ${colors.secondary};
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+
+    div.ant-card-cover {
+      width: 100%;
+      height: 50%;
+
+      div.ant-image {
+        height: 100%;
+
+        div.ant-image-img {
+          height: 100%;
+          border-radius: 0.5rem;
+        }
+
+        div.ant-image-mask {
+          border-radius: 0.5rem;
+        }
+      }
+
+      img {
+        height: 100%;
+        border-radius: 0.5rem;
+      }
+    }
+
+    div.ant-card-body {
+      padding: 0.75rem 0.25rem 0.25rem;
+      height: 50%;
+      display: grid;
+      grid-template-rows: 0.5fr 1fr 2fr 1fr;
+      gap: 0.5rem;
+
+      &::after,
+      &::before {
+        display: none;
+        content: none;
+      }
+    }
   }
 `;
 
-const CustomText = styled.p`
-  margin: 10px;
-  font-size: 1rem;
+const CardHeader = styled.p`
+  font-size: 1.2rem;
+  font-weight: 700;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans',
+    'Helvetica Neue', sans-serif;
+  color: ${colors.text};
+  letter-spacing: 0.1px;
+  padding-left: 0.25rem;
+  line-height: 1.2;
+  display: block; /* Fallback for non-webkit */
+  display: -webkit-box;
+  -webkit-line-clamp: 1; /* no of lines */
+  text-overflow: ellipsis;
+  overflow: hidden !important;
+  -webkit-box-orient: vertical;
+`;
+
+const CardDescription = styled.span`
+  font-size: 0.75rem;
+  font-weight: 400;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans',
+    'Helvetica Neue', sans-serif;
+  opacity: 0.9;
+  padding-left: 0.25rem;
+  color: ${colors.text};
+  display: block; /* Fallback for non-webkit */
+  display: -webkit-box;
+  -webkit-line-clamp: 3; /* no of lines */
+  line-height: 1.4;
+  text-overflow: ellipsis;
+  overflow: hidden !important;
+  -webkit-box-orient: vertical;
 `;
 
-export function TracksCard({ collectionName, trackName, artworkUrl100: imgUrl /*previewUrl*/ }) {
+const PlayBtn = styled(Button)`
+  && {
+    background-color: ${colors.primaryLight};
+    border: none;
+    color: ${colors.secondary};
+    border-radius: 0.5rem;
+    font-size: 0.75rem;
+    font-weight: 500;
+    width: 100%;
+
+    &:hover {
+      background-color: ${colors.primaryDark};
+    }
+  }
+`;
+
+const BtnContainer = styled.div`
```

Comment:
can we change BtnContainer to ButtonContainer. Because at few places you use Btn and at few places it's Button. So lets change it everywhere to Button